src/cairoUtilFuncGen/storage/dynArrayPop.ts

```diff
@@ -90,8 +90,8 @@ export class DynArrayPopGen extends StringIndexedFuncGen {
     const getElemLoc =
       isDynamicArray(elementType) || isMapping(elementType)
         ? endent`
-            let elem_loc = ${arrayName}::read((loc, newLen));
-            let elem_loc = readId(elem_loc);
+            let elem_loc_id = ${arrayName}::read((loc, newLen));
+            let elem_loc = readId(elem_loc_id);
```

Comment:
nitpick, but I would go for:
* elem_id_loc  for the first one, since is the location of the id or pointer
* elem_id for the second one, since it holds the actual id